app/controllers/app.rb

```diff
@@ -21,36 +25,58 @@ class App < Roda
         view 'home'
       end
 
+      # GET /404
       routing.on '404' do
-        routing.get do
-          view '404'
-        end
+        view '404'
       end
 
+      # GET /resource_nil
       routing.on 'resource_nil' do
         view 'resource_nil'
       end
 
       # Inbox
       routing.on 'inbox' do
-        routing.on 'new' do
+        
+        # GET /inbox/new-inbox
+        routing.on 'new-inbox' do
           view 'new_inbox'
```

Comment:
This can return 'inbox'. Since we're being explicit about the view names, we don't need to create a view for each new route. Sometimes it just makes sense to use existing uses if they would do the same thing

Reply:
Done.